app/presenters/webauthn_setup_presenter.rb

```diff
@@ -32,6 +33,19 @@ def image_path
     end
   end
 
+  def learn_more_html
+    new_tab_link_to(
+      t('forms.webauthn_setup.learn_more'),
+      help_center_redirect_path(
+        category: 'get-started',
+        article: 'security-key',
+        article_anchor: 'security-key',
+        flow: :two_factor_authentication,
+        step: :security_key_setup,
+      ),
+    )
+  end
```

Comment:
I just wanted to double check - is it expected that this link will not work in local or in the [review app](https://review-lg-12596-u-o99u5a.review-app.identitysandbox.gov/) because of the nature of redirects to the help center? Live testing has it redirecting to the /account page.
http://localhost:3000/redirect/help_center?article=security-key&article_anchor=security-key&category=get-started&flow=two_factor_authentication&step=security_key_setup

Reply:
This is a good catch, I changed too many keys when moving to a presenter to try and get the anchor to work.

For a similar link, on TOTP setup, we see the following code:

```
<%= new_tab_link_to(
      t('links.what_is_totp'),
      help_center_redirect_path(
        category: 'get-started',
        article: 'authentication-options',
        article_anchor: 'authentication-application',
        flow: :two_factor_authentication,
        step: :totp_setup,
      ),
    ) %>
```

The anchor part for that doesn't seem to work.

I've pushed a quick change for security key that should link to the article.

Reply:
That works good now 👍